yarn.lock

```diff
@@ -9802,11 +9798,16 @@ tslib@^1.10.0, tslib@^1.8.1:
   resolved "https://registry.yarnpkg.com/tslib/-/tslib-1.14.1.tgz#cf2d38bdc34a134bcaf1091c41f6619e2f672d00"
   integrity sha512-Xni35NKzjgMrwevysHTCArtLDpPvye8zV/0E4EyYn43P7/7qvQwPh9BGkHewbMulVntbigmcT7rdX3BNo9wRJg==
 
-tslib@^2.0.0, tslib@^2.5.0, tslib@^2.6.2:
+tslib@^2.0.0:
```

Comment:
is this duplicate with `tslib@^2.6.2` below 9806?

Reply:
Yes. I don't know why `yarn` is doing that, but I do know we aren't supposed to edit the `yarn.lock` file by hand, for it is quick to anger and slow to forgive.